yarn.lock

```diff
@@ -120,11 +120,6 @@ ansi-escapes@^4.2.1:
   dependencies:
     type-fest "^0.11.0"
 
-ansi-regex@^2.0.0:
```

Comment:
i am not 100% sure if yarn is they way to go anymore, npm has done some improvements in the past. What do you think? Should we switch from yarn to npm lock files?

Reply:
Just opened an issue: https://github.com/nodefluent/node-sinek/issues/155 to discuss this further

Reply:
Roger